modules/cnf-getting-the-dpll-firmware-version-for-intel-e810-nics.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-getting-the-dpll-firmware-version-for-intel-e810-nics_{context}"]
+= Getting the DPLL firmware version for the CGU in Intel e810 NICs
+
+Get the digital phase-locked loop (DPLL) firmware version for the Clock Generation Unit (CGU) in Intel e810 NICs by opening a debug shell to the cluster node where you have installed the NIC and querying the NIC hardware.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+*  You have installed the PTP Operator on a bare-metal cluster with hosts that support PTP.
+
+* You have installed an Intel e810 NIC in a cluster host.
+
+.Procedure
+
+. Start a debug pod by running the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
++
+where:
++
+--
+<node_name> :: Is the node where you have installed the e810 NIC.
+--
+
+. Check the CGU firmware version in the NIC by using the `devlink` tool and the bus and device name where the NIC is installed.
+For example, run the following command:
++
+[source,terminal]
+----
+sh-4.4# devlink dev info <bus_name>/<device_name> | grep cgu
+----
++
+where:
++
+--
+<bus_name> :: Is the bus where the NIC is installed. For example, `pci`.
+<device_name> :: Is the NIC device name. For example, `0000:51:00.0`.
```

Comment:
If you want you can add this details , sine the output is very cryptic . 
evlink dev info pci/0000:51:00.0 | grep cgu
         [cgu.id](http://cgu.id/) 36
        fw.cgu 8032.16973825.6201
 DPLL Firmware is 6201 and DPLL Model is 8032 and DPLL config version is `16973825`.
 DPLL Configuration version (which is 1.3.0.1 ): 16973825 in binary is: 0001 0000 0011 0000 0000 0000 0001 where each byte is the part of the version.
 0001 -> 1, 0000 0011 -> 3, 0000 0000 -> 0,   0000 0001 -> 1.